### DISCUSSION TITLE
File edits take 3 to 4 mins to get refreshed in raw format

### DISCUSSION THREAD

**Discussion by RajeshSivadasan** [community/community#40704] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If a file/code is edited and committed on the github portal, it is not refreshed instantly in the raw format. It takes 3 to 4 mins to get the updated information refreshed.

**Comment by ossanna16** [comment]
Hi there @RajeshSivadasan and welcome to our community! Thank you for reporting this!
